pkg/internal/venafi/sign.go

```diff
@@ -38,6 +38,8 @@ func (err ErrCustomFieldsType) Error() string {
 	return fmt.Sprintf("certificate request contains an invalid Venafi custom fields type: %q", err.Type)
 }
 
+var ErrorMissingSubject = errors.New("certificate needs a comon name or at least one field in the subject set to be processed by Venafi")
```

Comment:
`Certificate requests submitted to Venafi issuers must have the 'commonName' field or at least one other subject field set.`

Reply:
Changed this